components/FAQ/Faq.tsx

```diff
@@ -0,0 +1,23 @@
+import { MdExpandMore } from 'react-icons/md'
+import { useState } from 'react'
+import styles from './faq.module.css'
+import FaqArray from './faqArray'
+import FaqItem from './FaqItem'
+
+const Faq = () => {
+  return (
+    <div style={{ height: '2000px' }} className="h-max">
+      {FaqArray.map((faqItem) => {
+        return (
+          <FaqItem
+            key={faqItem.question}
+            question={faqItem.question}
+            answer={faqItem.answer}
+          />
+        )
+      })}
+    </div>
+  )
+}
+
+export default Faq
```

Comment:
can you export it in line with the declaration just to keep consistent with the project

Reply:
i know some components are exported at the bottom..that's our bad but we've been meaning to export it in line for everything